lib/phoenix_live_view/channel.ex

```diff
@@ -613,6 +615,19 @@ defmodule Phoenix.LiveView.Channel do
     {:stop, :shutdown, :no_session}
   end
 
+  defp check_version_match(params) do
+    server_version = Application.spec(:phoenix_live_view, :vsn) |> List.to_string()
+    client_version = Map.get(params, "lv_version", :not_found)
+    should_display_warning = Application.get_env(:phoenix_live_view, :warn_version_mismatch, true)
```

Comment:
You can provide a default value for the environment by adding `env: [warn_version_mismatch: true]` here:

https://github.com/phoenixframework/phoenix_live_view/blob/master/mix.exs#L35

And then here you can do:

```suggestion
    should_display_warning = Application.fetch_env!(:phoenix_live_view, :warn_version_mismatch)
```